packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<header>
+		<b>{{ initialId }}</b>
+		<span v-if="name" class="name">{{ '| ' + name }}</span>
+		<span v-if="unit" class="unit">({{ unit }})</span>
+		<span v-if="description" class="description">{{ description }}</span>
+	</header>
+	<main>
+		<span class="expression">
+			<tera-input
+				label="Expression"
+				:model-value="getInitialExpression(modelConfiguration, initialId)"
+				@update:model-value="emit('update-expression', { id: initialId, value: $event })"
```

Comment:
Why are you sending an event?
It's all reactive, so if you update the modelConfiguration here, the rest will update directly?

Reply:
since modelConfiguration is a prop would it not be bad practice to manipulate it with a child component?  I send the event up to the parent where modelConfiguration prop is defined to manipulate it there

Reply:
... you are right, but it's bloody annoying.